### PR TITLE
Add list of policies for each version

### PR DESCRIPTION
**Pull request by davidor**
With this, [Porta](https://github.com/3scale/porta) will be able to show a different list of policies based on the version of APIcast deployed.

### CODE REVIEW

File: doc/policies_list/3.3.0/policies.json [hunk 1]
```diff
@@ -0,0 +1,1276 @@
...
+            "batch_report_seconds": {
+              "description": "Duration (in seconds) for batching reports",
+              "type": "integer"
+            }
+          },
+          "type": "object"
+        },
+        "description": [
+          "This policy caches authorizations from the 3scale backend ",
+          "and also reports in batches. Doing this is more efficient than ",
```
Comment (porueesq) [inline doc/policies_list/3.3.0/policies.json]: _and reports them in batches. This is more efficient than_

File: doc/policies_list/3.3.0/policies.json [hunk 2]
```diff
@@ -0,0 +1,1276 @@
...
+      }
+    ],
+    "3scale_referrer": [
+      {
+        "$schema": "http://apicast.io/policy-v1/schema#manifest#",
+        "configuration": {
+          "properties": {},
+          "type": "object"
+        },
+        "description": "Sends the 'Referer' to 3scale backend so it can be validated",
```
Comment (porueesq) [inline doc/policies_list/3.3.0/policies.json]: _Sends the 'Referer' to 3scale backend for validation_

File: doc/policies_list/3.3.0/policies.json [hunk 3]
```diff
@@ -0,0 +1,1276 @@
...
+        "name": "3scale Referrer",
+        "summary": "Sends the 'Referer' to 3scale backend so it can be validated.",
```
Comment (porueesq) [inline doc/policies_list/3.3.0/policies.json]: _Sends the 'Referer' to 3scale backend for validation_

File: doc/policies_list/3.3.0/policies.json [hunk 4]
```diff
@@ -0,0 +1,1276 @@
...
+                }
+              ],
+              "type": "string"
+            }
+          },
+          "type": "object"
+        },
+        "description": [
+          "Configures a cache for the authentication calls against the 3scale ",
+          "backend. This policy support four kinds of caching: \n",
```
Comment (porueesq) [inline doc/policies_list/3.3.0/policies.json]: _... This policy supports four kinds of caching:_

File: doc/policies_list/3.3.0/policies.json [hunk 5]
```diff
@@ -0,0 +1,1276 @@
...
+          " - Strict: it only caches authorized calls. Denied and failed calls ",
+          "invalidate the cache entry.\n",
+          " - Resilient: caches authorized and denied calls. Failed calls do not ",
+          "invalidate the cache. This allows us to authorize and deny calls ",
```
Comment (porueesq) [inline doc/policies_list/3.3.0/policies.json]: @davidor who is "us" in _This allows us to authorize and deny calls_? For example, does it refer to 3scale AMP?

File: doc/policies_list/3.3.0/policies.json [hunk 6]
```diff
@@ -0,0 +1,1276 @@
...
+          "according to the result of the last request made even when backend is ",
+          "down.\n",
+          "- Allow: caches authorized and denied calls. When backend is ",
+          "unavailable, it will cache an authorization. In practice, this means ",
+          "that when backend is down _any_ request will be authorized unless last ",
+          "call to backend for that request returned 'deny' (status code = 4xx). ",
+          "Make sure to understand the implications of that before using this ",
```
Comment (porueesq) [inline doc/policies_list/3.3.0/policies.json]: _Make sure to understand the implications of this Caching mode before using it._

File: doc/policies_list/3.3.0/policies.json [hunk 7]
```diff
@@ -0,0 +1,1276 @@
...
+          "mode. It makes sense only in very specific use cases.\n",
```
Comment (porueesq) [inline doc/policies_list/3.3.0/policies.json]: You can remove this line. The previous one is strong enough.

File: doc/policies_list/3.3.0/policies.json [hunk 8]
```diff
@@ -0,0 +1,1276 @@
...
+              ],
+              "type": "object"
+            }
+          },
+          "properties": {
+            "condition": {
+              "$ref": "#/definitions/condition"
+            },
+            "policy_chain": {
+              "description": "the policy chain to execute when the condition is true",
```
Comment (porueesq) [inline doc/policies_list/3.3.0/policies.json]: We need to have a more consistent capitalization in the UI. Can you please review descriptions like this to ensure they begin with a capital letter?
_The policy chain to execute when the condition is true_

File: doc/policies_list/3.3.0/policies.json [hunk 9]
```diff
@@ -0,0 +1,1276 @@
...
+            }
+          },
+          "required": [
+            "condition"
+          ],
+          "type": "object"
+        },
+        "description": [
+          "Evaluates a condition, and when it's true, it calls its policy chain. ",
+          "This policy cannot be configured from the 3scale UI."
```
Comment (porueesq) [inline doc/policies_list/3.3.0/policies.json]: Does _3scale UI_ refer to the Admin Portal? If so, replace _3scale UI_ with _Admin Portal_.

File: doc/policies_list/3.3.0/policies.json [hunk 10]
```diff
@@ -0,0 +1,1276 @@
...
+        "version": "builtin"
+      }
+    ],
+    "cors": [
+      {
+        "$schema": "http://apicast.io/policy-v1/schema#manifest#",
+        "configuration": {
+          "properties": {
+            "allow_credentials": {
+              "description": "Whether the request can be made using credentials",
```
Comment (porueesq) [inline doc/policies_list/3.3.0/policies.json]: It looks like something is missing here: _This policy checks whether the request can be made using credentials._
I'm assuming this is a policy; if not, replace the term "policy" with a more appropriate one.

File: doc/policies_list/3.3.0/policies.json [hunk 11]
```diff
@@ -0,0 +1,1276 @@
...
+            },
+            "allow_origin": {
+              "description": "Origins for which the response can be shared with",
+              "type": "string"
+            }
+          },
+          "type": "object"
+        },
+        "description": [
+          "This policy enables CORS (Cross Origin Resource Sharing) request ",
```
Comment (porueesq) [inline doc/policies_list/3.3.0/policies.json]: _This policy enables Cross Origin Resource Sharing (CORS) request_

File: doc/policies_list/3.3.0/policies.json [hunk 12]
```diff
@@ -0,0 +1,1276 @@
...
+              "type": "string"
+            }
+          },
+          "required": [
+            "auth_type"
+          ],
+          "type": "object"
+        },
+        "description": [
+          "This policy allows to expose a service without authentication. ",
```
Comment (porueesq) [inline doc/policies_list/3.3.0/policies.json]: _This policy allows service exposure without authentication._

File: doc/policies_list/3.3.0/policies.json [hunk 13]
```diff
@@ -0,0 +1,1276 @@
...
+          "It can be useful, for example, for legacy apps that cannot be adapted to ",
+          "send the auth params. ",
+          "When the credentials are not provided in the request, this policy ",
+          "provides the default ones configured. ",
+          "An app_id + app_key or a user_key should be configured."
```
Comment (porueesq) [inline doc/policies_list/3.3.0/policies.json]: _You need to configure user_key; or, the combination of app_id + app_key _

File: doc/policies_list/3.3.0/policies.json [hunk 14]
```diff
@@ -0,0 +1,1276 @@
...
+    ],
+    "liquid_context_debug": [
+      {
+        "$schema": "http://apicast.io/policy-v1/schema#manifest#",
+        "configuration": {
+          "properties": {},
+          "type": "object"
+        },
+        "description": [
+          "This is a policy meant only for debugging purposes. This policy ",
```
Comment (porueesq) [inline doc/policies_list/3.3.0/policies.json]: _This is a policy intended only for debugging purposes..._

File: doc/policies_list/3.3.0/policies.json [hunk 15]
```diff
@@ -0,0 +1,1276 @@
...
+          "returns the context available when evaluating liquid. Any policy can ",
+          "modify the context that is shared between policies and that context is ",
+          "available when evaluating liquid. However, documenting what's available ",
```
Comment (porueesq) [inline doc/policies_list/3.3.0/policies.json]: _...documenting what is available_

We avoid using contractions.

File: doc/policies_list/3.3.0/policies.json [hunk 16]
```diff
@@ -0,0 +1,1276 @@
...
+                },
+                "type": "object"
+              },
+              "type": "array"
+            }
+          },
+          "type": "object"
+        },
+        "description": [
+          "Captures arguments in a URL and rewrites the URL using those arguments. ",
```
Comment (porueesq) [inline doc/policies_list/3.3.0/policies.json]: _Captures arguments in a URL and rewrites the URL using these arguments._

File: doc/policies_list/3.3.0/policies.json [hunk 17]
```diff
@@ -0,0 +1,1276 @@
...
+          "For example, we can specify a matching rule with arguments like ",
+          "'/{orderId}/{accountId}' and a template that specifies how to rewrite ",
+          "the URL using those arguments, for example: ",
```
Comment (porueesq) [inline doc/policies_list/3.3.0/policies.json]: _the URL using those arguments. Example:_

File: doc/policies_list/3.3.0/policies.json [hunk 18]
```diff
@@ -0,0 +1,1276 @@
...
+          "'/sales/v2/{orderId}?account={accountId}'. In that case, the request ",
```
Comment (porueesq) [inline doc/policies_list/3.3.0/policies.json]: _'/sales/v2/{orderId}?account={accountId}'; in this case, the request _

**Comment by mikz** [comment]
@porueesq those are already released policies. We are not going to change those descriptions. 

Feel free to open PRs to change any of the policy schemas on the master branch: https://github.com/3scale/APIcast/search?q=filename%3Aapicast-policy.json&unscoped_q=filename%3Aapicast-policy.json

**Comment by porueesq** [comment]
@mikz @davidor Given that this is not the first time that happens, what's the purpose of requesting reviews from the Docs team if you are merging the changes before we have time to send the comments? Perhaps if you let us know with in advance, we can organize our tasks to provide you a better service.

**Comment by davidor** [comment]
@porueesq I will merge your changes in a separate PR. We cannot change the schemas of previous versions. However, The ones that you reviewed still exist, so we will merge those changes and the fixed descriptions will appear in the UI in the next release.

**Comment by porueesq** [comment]
Thanks @davidor :)

**Comment by mikz** [comment]
@porueesq the requested review is automatic because you are set to be the code owner of `doc` folder of the repo. https://github.com/3scale/APIcast/blob/master/.github/CODEOWNERS

**Comment by davidor** [comment]
@porueesq https://github.com/3scale/APIcast/pull/1020

**Comment by porueesq** [comment]
@mikz Great to know that we are involved! 
What I've seen is that a review is requested, we comment, and then we are told that our comments won't be applied. 
We just need some time to review the content to provide relevant contributions. In case you have time constraints, please let us know.

**Pull request merged**